src/main/java/seedu/homerce/logic/commands/UndoCommand.java

```diff
@@ -0,0 +1,25 @@
+package seedu.homerce.logic.commands;
+
+import seedu.homerce.logic.commands.exceptions.CommandException;
+import seedu.homerce.model.HistoryManager;
+import seedu.homerce.model.Model;
+
+/**
+ * Undo the last command that updates the state of Homerce's storage.
+ *
+ * A command that does not affect the state of Homerce's storage will not be considered in the undo command.
+ */
+public class UndoCommand extends Command {
+    public static final String COMMAND_WORD = "undo";
+    public static final String MESSAGE_SUCCESS = "You have undone you previous change!";
```

Comment:
Message takes a bit of time to understand for me, maybe undo success or undo complete is sufficient 😸